### PR TITLE
Clickable username

### PR DESCRIPTION
**Pull request by kieter**
Sad. There's conflicts.

### CODE REVIEW

**Comment by kieter** [comment]
Btw, once this pull request is accepted. Please use my TextView (called UsernameTextView) for any usernames.

File: app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java [hunk 1]
```diff
@@ -28,13 +28,19 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_user_profile);
+
+        // Retrieve the user this intent was started with.
+        Bundle bundle = getIntent().getExtras();
+        User user = bundle.getParcelable("user");
+
```
Comment (Hreherch) [inline app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java:35]: Make sure this doesn't break viewing a profile normally (i.e. through the MainActivity "view profile" button).

Comment (Hreherch) [reply app/src/main/java/comcmput301f16t01/github/carrier/UserProfileActivity.java:35]: We should also consider if we want to always pull from elastic search when we do this activity. To get the most up to date information (but blocks the user from getting information while they are offline)

**Review by Hreherch — CHANGES_REQUESTED** [review]
java.lang.IllegalStateException: Could not find method onClickDriverUsernameTextView(View) in a parent or ancestor Context for android:onClick attribute defined on view class android.widget.LinearLayout

**Review by Hreherch — APPROVED** [review]

**Pull request merged**